auth/api/metrics.go

```diff
@@ -152,12 +152,12 @@ func (ms *metricsMiddleware) AddPolicies(ctx context.Context, prs []auth.PolicyR
 	return ms.svc.AddPolicies(ctx, prs)
 }
 
-func (ms *metricsMiddleware) DeletePolicy(ctx context.Context, pr auth.PolicyReq) error {
+func (ms *metricsMiddleware) DeletePolicyFilter(ctx context.Context, pr auth.PolicyReq) error {
```

Comment:
Change the name of metric being sent

Reply:
@nyagamunene please address this comment